### PR TITLE
Fixes double pixel border on bottom of citation

### PR DESCRIPTION
**Pull request by Ghalko**
## Purpose

Removes the bottom border of the citation addon header when the box is collapsed.

screen shot 2015-02-23 at 12 22 31 pm

With the double border

## Changes

screen shot 2015-02-23 at 12 24 25 pm

Without the double border. Much better!

screen shot 2015-02-23 at 12 29 13 pm

Border returns when expanded.

Added .addon-widget-header.hide-border to classes and css. Used it in the javascript to remove the border.

### CODE REVIEW

**Comment by lbanner** [comment]
PR #2? .... :smiling_imp:

**Comment by sloria** [comment]
Instead of adding another CSS class and applying it with JS, could we just fix the `addon-widget-header`? ping @caneruguz for input on this.

**Comment by caneruguz** [comment]
I agree with Steve, I think we should just remove the bottom border from the header.

**Comment by Ghalko** [comment]
I could definitely do that. It does look a bit different.

**Comment by sloria** [comment]
@Ghalko screenshot?

**Comment by Ghalko** [comment]
screen shot 2015-03-06 at 10 26 31 am

It blends a bit without a definite border. Not sure if there is a paper out there about this phenomenon. But it would be a simpler fix.

**Comment by caneruguz** [comment]
Another option could be to put border-top to panel body, because the body gets hidden it would not have double border. 



—

**Comment by Ghalko** [comment]
True, let me see if I can do that.

File: website/static/css/site.css [hunk 1]
```diff
@@ -1935,9 +1935,11 @@ span#profileFullname{
     border:1px solid #ddd;
     margin-bottom: 20px;
 }
+.addon-widget-header.hide-border {
```
Comment (sloria) [inline website/static/css/site.css]: Is this used anywhere?

**Comment by sloria** [comment]
Pass finished. :pager:

**Pull request merged**